cegs_portal/search/tsv_templates/v1/non_targeting_reos.py

```diff
@@ -1,11 +1,42 @@
+from cegs_portal.search.templatetags.custom_helpers import format_significance
+
+
 def non_targeting_regulatory_effects(data):
     reo_page, feature = data
     tsv_data = []
-    tsv_data.append(['Chromosome Location', 'Effect Size', 'Direction', 'Significance', 'Distance', 'Feature Type', 'Experiment'])
+    tsv_data.append(
+        [
+            "chrom",
+            "chromStart",
+            "chromEnd",
+            "name",
+            "strand",
+            "score",
+            "Effect Size",
+            "Direction",
+            "Significance",
+            "Distance",
+            "Feature Type",
+            "Experiment",
```

Comment:
We should add another header here for the reo accession id column (explained below). Maybe called `effect id`

Reply:
Completed and will update other branch